### ISSUE TITLE
Consistent sideset and nodeset naming

### ISSUE THREAD

**Issue by lindsayad** [libMesh/libmesh#736]
If a nodeset is automatically created when a sideset is created (or visa versa), it would be nice if both objects received the same boundary_name. Currently, if I view a mesh in Paraview for example, I can see the properly named sideset ('left' for instance), but the corresponding nodeset that was created does not share the name (instead it's something like "Unnamed st ID: 3"). This doesn't have much to do with functionality; I believe I can still access the nodeset using the boundary_name I gave to the sideset. However, it would be nice from an aesthetic (and organizational) point-of-view.

**Comment by permcody** [comment]
:+1:

**Comment by dschwen** [comment]
and this is whats breaking Paraview 5.10 for us now :-D

**Comment by lindsayad** [comment]
That they don’t have the same name breaks paraview? Wow I made this issue when I was still in grad school. Blast from the past!

**Comment by roystgnr** [comment]
That they *do* have the same name breaks the new Paraview Exodus reader (in a 5.10 RC and at least one of the 5.9 point releases?).  You have to load a module for the Legacy exodus reader and then select that when opening the file (not re-opening, if it's in your Recent list from a failed open).  Painful.  Looks like the problem has been fixed in the library their new reader relies on, though, so hopefully the fix will percolate up to them before the final 5.10 release.

**Comment by lindsayad** [comment]
Right, I was thinking that when we `build_side_list_from_node_list` (and visa versa) that we assign them the same name, so maybe as a libMesh issue this should actually be closed? I think this is actually more of a user code issue now...

**Comment by roystgnr** [comment]
Yeah, as far as I know this is a concern at the Moose level.  libMesh never automatically creates a sideset when a nodeset exists or vice-versa, we just provide tools to make it easier for user code to do so if desired.
